ec2/tests/test_ec2_elasticip.py

```diff
@@ -0,0 +1,236 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+# Built-in Imports
+import testtools
+
+# Third Party Imports
+from moto import mock_ec2
+
+# Cloudify Imports is imported and used in operations
+from ec2 import connection
+from ec2 import utils
+from ec2 import elasticip
+from cloudify.mocks import MockCloudifyContext
+from cloudify.mocks import MockContext
+from cloudify.exceptions import NonRecoverableError
+
+TEST_AMI_IMAGE_ID = 'ami-e214778a'
+TEST_INSTANCE_TYPE = 't1.micro'
+FQDN = '((?:[a-z][a-z\\.\\d\\-]+)\\.(?:[a-z][a-z\\-]+))(?![\\w\\.])'
+
+
+class TestElasticIP(testtools.TestCase):
+
+    def mock_ctx(self, test_name):
+
+        test_node_id = test_name
+        test_properties = {
+            'image_id': TEST_AMI_IMAGE_ID,
+            'instance_type': TEST_INSTANCE_TYPE,
+            'attributes': {
+                'security_groups': ['sg-73cd3f1e'],
+                'instance_initiated_shutdown_behavior': 'stop'
+            }
+        }
+
+        ctx = MockCloudifyContext(
+            node_id=test_node_id,
+            properties=test_properties
+        )
+
+        return ctx
+
+    def mock_relationship_ctx(self, testname):
+
+        instance_ctx = MockContext({
+            'node': MockContext({
+                'properties': {}
+            }),
+            'instance': MockContext({
+                'runtime_properties': {
+                    'instance_id': 'i-abc1234'
+                }
+            })
+        })
+
+        elasticip_ctx = MockContext({
+            'node': MockContext({
+                'properties': {}
+            }),
+            'instance': MockContext({
+                'runtime_properties': {
+                    'elasticip': ''
+                }
+            })
+        })
+
+        relationship_ctx = MockCloudifyContext(node_id=testname,
+                                               source=instance_ctx,
+                                               target=elasticip_ctx)
+
+        return relationship_ctx
+
+    def test_create(self):
+        """
+        """
+        ctx = self.mock_ctx('test_create_address')
+
+        with mock_ec2():
+            elasticip.create(ctx=ctx)
+            self.assertTrue('elasticip' in ctx.instance.runtime_properties)
```

Comment:
should use assertIn rater than assertTrue(x in y). assertTrue only returns true or false while assertIn returns the exception.